.github/workflows/build-cli.yml

```
@@ -87,6 +87,7 @@ jobs:
         os:
          [
           { base: "linux", distro: "ubuntu-22.04", target_suffix: "" },
+          { base: "linux", distro: "ubuntu-22.04", target_suffix: "-arm64", runner: "ubuntu-22.04-arm" },
```

Since distro is only used in `runs-on`, can this be simplified to set the distro to `ubuntu-22.04-arm` and remove the runner part, or does it need to be split up?

For example:
```suggestion
          { base: "linux", distro: "ubuntu-22.04-arm", target_suffix: "-arm64" },
```

---

thanks for the suggestion! this is much cleaner
committed and validated the builds were successful